src/plugins/tableLoop/tableLoopPlugin.ts

```
@@ -0,0 +1,131 @@
+        this.utilities = utilities;
+        this.loopStrategies.forEach(strategy => strategy.setUtilities(utilities));
+    }
+
+    public async containerTagReplacements(tags: Tag[], data: ScopeData, context: TemplateContext): Promise<void> {
+
+        let value = data.getScopeData<TemplateData[]>();
+
+        // Non array value - treat as a boolean condition.
+        const isCondition = !Array.isArray(value);
+        if (isCondition) {
+            if (!!value) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)